lib/index.js

```diff
@@ -320,7 +337,7 @@ internals.redirectMethod = function (code, method, options) {
 
 internals.Client.prototype.read = function (res, options, callback) {
 
-    options = Hoek.applyToDefaultsWithShallow(options || {}, this._defaults, internals.shallowOptions);
+    options = Hoek.applyToDefaultsWithShallow(this._defaults, options || {}, internals.shallowOptions);
```

Comment:
I was confused/surprised when I saw this! Please confirm this is correct? I believe without this change we're giving higher precedence to `.defaults` than the options passed into read itself